x/migration/fs_source_test.go

```diff
@@ -0,0 +1,45 @@
+// +build go1.16
+
+package migration
+
+import (
+	"context"
+	"testing"
+	"testing/fstest"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestFSSource(t *testing.T) {
+	var (
+		ctx = context.Background()
+		fs  = fstest.MapFS{
+			"3_final.down.cql": &fstest.MapFile{Data: []byte("bar")},
+			"2_initial.up.cql": &fstest.MapFile{Data: []byte("foo")},
+			"3_final.up.cql":   &fstest.MapFile{Data: []byte("bar")},
```

Comment:
nit: should we enforce that the file is a `.cql` ?
Otherwise same comment as for https://github.com/upfluence/sql/pull/19